docs/setting-up-metabase.md

```diff
@@ -42,7 +42,14 @@ One last quick thing that you’ll have to decide is if it’s okay for us to co
 
 ![Usage data preferences](images/UsageData.png)
 
-If you’re ready to start using Metabase, go ahead and click **Next** and then **Take me to Metabase**.
+If you’re ready to start using Metabase, go ahead and click **Next**
+
+## Staying in touch
+At this point you are all set and ready to use Metabase.  Since we like keeping in touch with our friends we made it easy to sign-up to our newsletter (infrequent emails) with a single click!
```

Comment:
Even though its mentioned via copy in the screenshot, might be good to mention that we'll be using the list let users know about new versions, best practices, that kind of thing so they get a sense of the content.